locale/core.yml

```diff
@@ -259,6 +259,7 @@ core:
       mark_all_as_read_tooltip: => core.ref.mark_all_as_read
       title: => core.ref.notifications
       tooltip: => core.ref.notifications
+      mark_all_as_read_confirm_text: "Are you sure you want to mark all notifications as read?"
```

Comment:
To be consistent, this should probably be called `mark_all_as_read_confirmation`.

/cc @dcsjapan